### DISCUSSION TITLE
config/frontmatter/article/

### DISCUSSION THREAD

**Discussion by giscus** [pengzhanbo/vuepress-theme-plume#269] [docs-comment]
# config/frontmatter/article/

https://theme-plume.vuejs.press/config/frontmatter/article/

**Comment by zhengchen-zzZ** [comment]
作者，博客目录和博客文章比较多的情况下，在博客分类展示默认是展开状态导致检索特定博客有点繁琐且困难（要滚动和手动折叠目录检索特定博客文章）。
因为现在默认是全部展开展示全部内容的，提议可否增加机制支持自定义显示的深度（比如只展示根分类目录或者展示到二级目录等 0 -> n 的配置项）

**Reply by pengzhanbo** [reply]
可以实现，你先提一个 feature request 的 issue 吧。

**Reply by zhengchen-zzZ** [reply]
博主，另外还有一处小瑕疵，就是同一级的文件夹下同时有md文件和文件夹，展示会出现md文件穿插在这些文件夹之间展示：
- folder1
   - folder2
   - file1.md //这里md文件集中穿插在分类中
   - file2.md
   - file3.md
   -  folder3
   -              //=>预期中md文件出现的位置
- folder4
- folder5
就是稍微有点强迫症，可否同时添加这个类似排序的功能

**Reply by pengzhanbo** [reply]
分类默认是按照时间排序的，我认为这里不需要支持自定义排序，有点怪异。 
但我会把 folder 统一提到 file 之前，这本来是我的预期实现，但现在看来没有实现预期功能。

**Reply by zhengchen-zzZ** [reply]
嗯，主要是文件在分类文件夹之间展示有些难受

**Comment by lhhh666** [comment]
您好，我想问下这个tags怎么筛选呢，比如我有a，b，c三个tag，跳到标签页的时候，我只想展示a和b这种情况

**Reply by pengzhanbo** [reply]
标签页会收集博客文章中的所有 tags。 标签页只支持单独展示某个标签下的文章集合，暂不支持多选。

**Comment by limou3434** [comment]
这个 `sticky` 字段是怎么使用的？我查看这个文档在 `.md` 文件中后书写了以下内容：
```md
---
title: xxxx
sticky: 3
createTime: 2024/06/08 13:05:15
permalink: /article/5xbd5qgx/
---
```
但是为什么不起效果呢？这个功能难道不是指自定义文件在分类中的顶置顺序么？

**Reply by limou3434** [reply]
默认按照时间排序的话，有一个问题很奇怪，如果我的文章在逻辑上具有从浅到深的逻辑，那么从用户视角上很奇怪——用户需要倒着切换下一篇文章...

**Reply by limou3434** [reply]
噢我理解错这个字段的意思了，原来是在点击博客菜单页面后中的顶置...分类中的文件无法排序么？

**Reply by pengzhanbo** [reply]
默认是按照文件命名的编码进行排序。直接在文件 [名称前添加序号](https://theme-plume.vuejs.press/guide/write/#%E7%BA%A6%E5%AE%9A)。
